plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py

```diff
@@ -46,36 +49,48 @@ def get_excluded_ports(self):
         return '\n'.join(formatted_items) + ('' if not formatted_items else '\n')
 
 
-    def exclude_ports(self, excluded_ports_str):
+    def exclude_ports(self):
         """
         Parse input ports and add them to exclude list (or just update TTL)
         Input string example: 0c42a10300756a04_1,98039b03006c73ba_2:300
```

Comment:
that's not the case anymore if I'm not mistaken

Reply:
Fixed

Reply:
Fixed